includes/utils/class-amp-validation-utils.php

```diff
@@ -483,4 +487,62 @@ public static function add_validation_callback( $sanitizers ) {
 		return $sanitizers;
 	}
 
+	/**
+	 * Registers the post type to store the validation errors.
+	 *
+	 * @return void.
+	 */
+	public static function post_type() {
```

Comment:
Let's keep method names verby. So this should be `register_post_type`

Reply:
Thanks, [this commit](https://github.com/Automattic/amp-wp/commit/4955686933b6aaef1807cc1e1901b8a52c438415) renames it to `register_post_type()`.